src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
+    readyStudentEditPage();
+});
+
+/*
+ * Sends confirmation of changes when team name box is edited
```

Comment:
<code>Sends confirmation</code> doesn't seem like the right word to describe it. Anyway, this function adds a event handler, so I think you should describe adding the handler instead.